lib/scrabble/scrabble.rb

```diff
@@ -0,0 +1,84 @@
+module Scrabble
+  class Scrabble
+
+    SCORING= {a: 1, b: 3, c: 3, d: 2, e: 1, f: 4,
+      g: 2, h: 4, i: 1, j: 8, k: 5,
+      l: 1, m: 1, n: 1, o: 1, p: 3,
+      q: 10, r: 1, s: 1, t: 1, u: 1,
+      v: 4, w: 4, x: 8, y: 4, z: 10}
+
+    def self.score(word)
+      #accounting for blank user input
+      return 0 if word == ""
+      return 0 if word == nil
```

Comment:
There's a few options for this that would end up being a little bit tighter, and are all open to personal choice.

```ruby
return 0 if word.to_s.empty?
```

this will work because `nil.to_s` will return `""` and `empty?` works on strings only. Unfortunately, the string " " (a single space, or really any number of spaces) will return false, but we can use the Ruby string method `strip` to remove spaces from the end of any string, so if the user somehow enters "       " this would catch it:

```ruby
return 0 if word.to_s.strip.empty?
```

Another way to do this that you might see is like this:

```ruby
return 0 if ["", nil].includes? word
```
here we're checking whether or not the array includes the value of word. Its a nice variation because it shows you clearly what its checking for or concerned about coming in from the user.